autouri/cli.py

```diff
@@ -14,6 +14,9 @@
 logger = logging.getLogger(__name__)
 
 
+DEFAULT_RMDIR_NTH = 6
```

Comment:
Seems like a redefinition of `DEFAULT_NUM_THREADS` in `URIBase`, would be good to keep the constant in one place.

Reply:
That's a good point. Replaced it with `URIBase.DEFAULT_NUM_THREADS`.